prog-o-meter.py

```diff
@@ -215,6 +252,9 @@ def update_log():
         save.pack(side=Tk.LEFT, expand=True)
         clear.pack(side=Tk.LEFT, expand=True)
         close.pack(side=Tk.LEFT, expand=True)
+    def open_personalization(self):
+        self.root.destroy()

```

Comment:
It closes the progress GUI window and opens the personalization one. It is needed to close the window before opening another one because the other tkinter window will occupy the entire process and other things done after it is opened will run only when it is closed.

Reply:
Thanks, that makes sense. Maybe we should just leave a comment here about why we close the window (mainly because in most other cases in this program we do not close the lower level window, but just open a top-level window on the top).